### PR TITLE
GB-18 - Implement social auth via google connect

### PR DESCRIPTION
**Pull request by veranika107**

### CODE REVIEW

**Comment by veranika107** [comment]
[GB - Implement social auth via google connect](https://trello.com/c/XRdxRwyK/18-gb-implement-social-auth-via-google-connect)

File: src/Controller/Api/v1/GoogleAuthenticationController.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+use Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use KnpU\OAuth2ClientBundle\Client\ClientRegistry;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use UnexpectedValueException;
+
+class GoogleAuthenticationController extends AbstractController
```
Comment (wechka53) [inline src/Controller/Api/v1/GoogleAuthenticationController.php:19]: ?

File: src/Service/Google/GoogleUserManager.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Service\Google;
+
+use App\Entity\User;
+use App\Repository\UserRepository;
+use League\OAuth2\Client\Provider\GoogleUser;
+use UnexpectedValueException;
+
+class GoogleUserManager
```
Comment (wechka53) [inline src/Service/Google/GoogleUserManager.php:10]: ?

**Review by wechka53 — APPROVED** [review]

File: src/Controller/Api/v1/GoogleAuthenticationController.php (hunk 1 above)
Comment (veranika107) [reply src/Controller/Api/v1/GoogleAuthenticationController.php:18]: changes are made. Thanks

File: src/Service/Google/GoogleUserManager.php (hunk 2 above)
Comment (veranika107) [reply src/Service/Google/GoogleUserManager.php:10]: changes are made. Thanks

**Pull request merged**